library/core/src/intrinsics/mod.rs

```diff
@@ -3242,6 +3242,8 @@ pub const fn bitreverse<T: Copy>(_x: T) -> T {
 /// large and difficult to optimize.
 ///
 /// The stabilized version of this intrinsic is [`Ord::cmp`].
+#[rustc_intrinsic_const_stable_indirect]
+#[rustc_nounwind]
 #[rustc_intrinsic]
 #[rustc_intrinsic_must_be_overridden]
```

Comment:
FWIW `rustc_intrinsic_must_be_overridden` does not exist any more, I guess your branch has an outdated base and that's why it shows up here? Might be worth a rebase.